ui/cryptomaterial/wallet_clickable_list.go

```diff
@@ -0,0 +1,116 @@
+package cryptomaterial
+
+import (
+	"gioui.org/layout"
+	"gioui.org/unit"
+	"github.com/crypto-power/cryptopower/libwallet/utils"
+)
+
+type ClickedItem struct {
```

Comment:
I think this can be confusing to use since we only use it for wallets/assets, maybe we can rename this to something like "WalletClickableListItem"

Reply:
`SelectedItem`, `SelectedAsset`